src/app/visit/visit.component.css

```diff
@@ -37,7 +37,19 @@
     cursor: pointer;
 }
 
+.loader-container {
+    position: relative;
+    min-height: 200px;
+    max-height: 400px;
+    overflow: auto;
+    background: rgba(0, 0, 0, 0.15);
+    height: 30vh;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
 .selectedRow,
 selectedRow:hover {
     background-color: lightgray !important;
-}
+}
```

Comment:
what do you have set that always remove the last empty line?

Reply:
done